electrum/plugins/ledger/ledger.py

```diff
@@ -688,20 +731,24 @@ def get_btchip_device(self, device):
     def create_client(self, device, handler):
         client = self.get_btchip_device(device)
         if client is not None:
-            client = Ledger_Client(client, product_key=device.product_key, plugin=self)
+            client = Ledger_Client(
+                client, product_key=device.product_key, plugin=self)
         return client
 
     def setup_device(self, device_info, wizard, purpose):
         device_id = device_info.device.id_
-        client = self.scan_and_create_client_for_device(device_id=device_id, wizard=wizard)
+        client = self.scan_and_create_client_for_device(
+            device_id=device_id, wizard=wizard)
         wizard.run_task_without_blocking_gui(
-            task=lambda: client.get_xpub("m/0'", 'standard'))  # TODO replace by direct derivation once Nano S > 1.1
+            task=lambda: client.get_xpub("m/2'", 'standard'))  # TODO replace by direct derivation once Nano S > 1.1
```

Comment:
FIND THIS!!!

Reply:
https://github.com/spesmilo/electrum/commit/ac239a81b8baec1d255efd39be8aed61ee04be90